dashboard/src/components/Tabs/Tests/StatusChart.tsx

```diff
@@ -0,0 +1,99 @@
+import { memo, useMemo } from 'react';
+
+import type { IBaseCard } from '@/components/Cards/BaseCard';
+import BaseCard from '@/components/Cards/BaseCard';
+
+import type { StatusChartValues } from '@/components/StatusChart/StatusCharts';
+import StatusChartMemoized, {
+  Colors,
+} from '@/components/StatusChart/StatusCharts';
+import { groupStatus } from '@/utils/status';
+import type { TTreeTestsData } from '@/types/tree/TreeDetails';
+
+interface IStatusChart extends Pick<TTreeTestsData, 'statusCounts'> {
+  title: IBaseCard['title'];
+}
+const StatusChart = ({ statusCounts, title }: IStatusChart): JSX.Element => {
+  const groupedStatusCounts = useMemo(
+    () =>
+      groupStatus({
```

Comment:
Could you make `groupStatus` treat null as 0? so that we don't need to use `status ?? 0` throughout the project